time stamp

float32[3] acc_data  # [m/s^2]
float32[3] gyro_data # [rad/s]
float32[3] mag_data  # [T]
float32[3] gravity # [m/s^2]

